src/clarity.rs

```diff
@@ -651,6 +690,30 @@ pub fn invoke_command(invoked_by: &str, args: &[String]) {
 #[cfg(test)]
 mod test {
     use super::*;
+    #[test]
+    fn test_initial_alloc() {
+        let db_name = format!("/tmp/db_{}", rand::thread_rng().gen::<i32>());
+        let json_name = format!("/tmp/test-alloc_{}.json", rand::thread_rng().gen::<i32>());
+        let clar_name = format!("/tmp/test-alloc_{}.clar", rand::thread_rng().gen::<i32>());
+
+        fs::write(&json_name, r#"
+[ { "principal": "S1G2081040G2081040G2081040G208105NK8PE5",
+    "amount": 1000 },
+  { "principal": "S1G2081040G2081040G2081040G208105NK8PE5.names",
+    "amount": 2000 } ]
+"#).unwrap();
+
+        fs::write(&clar_name, r#"
+(unwrap-panic (if (is-eq (stx-get-balance 'S1G2081040G2081040G2081040G208105NK8PE5) u1000) (ok 1) (err 2)))
+(unwrap-panic (if (is-eq (stx-get-balance 'S1G2081040G2081040G2081040G208105NK8PE5.names) u2000) (ok 1) (err 2)))
+"#).unwrap();
```

Comment:
@kantai How would a test for an account overflow look like?

Reply:
I think using this interface, it'd be a little hard to actually test a STX overflow, since that would require loading an account with u128::max_value, though JSON parsing really only works up to u64.